AZ3166/src/libraries/WebSocket/examples/WebSocketEcho/WebSocketEcho.ino

```diff
@@ -119,7 +119,7 @@ void loop()
         {
           recvResult = wsClient->receive(wsBuffer, sizeof(wsBuffer));
 
-          if (!recvResult && recvResult->length > 0)
+          if (recvResult && recvResult->length > 0)
```

Comment:
good catch, but it's totally not make sense that this test can work properly before -.-

Reply:
In previous work, the _recvResult_ is **NULL** but in devkit board pointer pointed to NULL value does not cause core dump since devkit don't have the security mechanism. In short, previous work was a fluke.